csharp/Perf/Microsoft.Spark.CSharp/PerfBenchmark.csproj

```diff
@@ -60,6 +71,10 @@
   </ItemGroup>
   <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent Condition=" '$(OS)' == 'Windows_NT' ">copy /y $(ProjectDir)..\..\..\csharp\Worker\Microsoft.Spark.CSharp\bin\$(ConfigurationName)\CSharpWorker.* $(TargetDir)</PostBuildEvent>
```

Comment:
Add(Copy) the `CSharpWorker.exe.config` to make it possible for the users to modify the log level of worker if they want.